src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -52,12 +53,14 @@ class Connection
     private $configuration = [];
     private $driverConnection;
     private $schemaSynchronizer;
+    private $schemaConfig;
 
     public function __construct(array $configuration, DBALConnection $driverConnection, SchemaSynchronizer $schemaSynchronizer = null)
     {
         $this->configuration = array_replace_recursive(self::DEFAULT_OPTIONS, $configuration);
         $this->driverConnection = $driverConnection;
         $this->schemaSynchronizer = $schemaSynchronizer ?? new SingleDatabaseSynchronizer($this->driverConnection);
+        $this->schemaConfig = $this->driverConnection->getSchemaManager()->createSchemaConfig();
```

Comment:
no need to assign this to a property. you can just inline it where it is used which makes it lazy, esp. with #32308